Sources/Sentry/SentryBinaryImageCache.m

```diff
@@ -35,8 +36,21 @@ - (void)stop
 
 - (void)binaryImageAdded:(const SentryCrashBinaryImage *)image
 {
+    if (image->name == NULL) {
+        SENTRY_LOG_WARN(@"The image name was NULL. Can't add image to cache.");
+        return;
+    }
+
+    NSString *imageName = [NSString stringWithCString:image->name encoding:NSUTF8StringEncoding];
```

Comment:
Are we sure `image->name` will never be NULL? That can crash this constructor.

Reply:
I added a check for that above `if (image->name == NULL)`. If it's `NULL` we return.